scipy/stats/__init__.py

```diff
@@ -114,6 +114,7 @@
    triang            -- Triangular
    truncexpon        -- Truncated Exponential
    truncnorm         -- Truncated Normal
+   truncweibull_min  -- Left truncated minimum Weibull distribution
```

Comment:
I was just trying to follow the convention "trunc" + "distribution name". Since the new distribution is the truncation of "weibull_min"... well, "truncweibull_min".

Anyway, I think the call is yours (scipy core dev team). I have no particular preference.

Reply:
I agree with that reasoning, @ace-e4s.